### ISSUE TITLE
MDS implementation with missing and censored values

### ISSUE THREAD

**Issue by adalisan** [acorg/Racmacs#175]
In the documentation, lack of an MDS implementation for this type of data is mentioned.  By "MDS", are you referring to metric MDS specifically, which assume the input matrix is a distance matrix that is based on a metric. There are relaxed non-metric variants of MDS (like ordinal MDS in smacof package) which would be more robust to outliers as well. I don't know whether the specific R implementation I mentioned support both missing and censored dissimilarities, but ordinal MDS in general should. The limitations of those variants are that they are more expensive to optimize usually.

**Comment by shwilks** [comment]
Yes, you're right, that question is about metric MDS specifically, but a non-metric variant of MDS that allows for both missing values and right-censored distances would be interesting, the smacof MDS options certainly seem quite comprehensive thanks for sharing that.

It would be nice to have a test set of different types of antigenic map data against which the performance of possible alternative approaches could be easily compared.

**Comment by adalisan** [comment]
I am relatively new to antigenic cartography, and have some experience in dissimilarity and embedding methodologies. As I understand it, there can be different levels of uncertainty and outliers in the source titer data. in comparing methods, robustness to  those may be more important than other factors (like computational efficiency). If there is no reason that you would expect to have the data to have a better fit for metric MDS, other approaches may work better overall.
